app/src/org/commcare/activities/EntitySelectSearchUI.java

```diff
@@ -93,9 +94,7 @@ public void setupActionImage(Callout callout) {
     private void restoreLastQuery() {
         String lastQueryString = activity.getLastQueryString();
         if (lastQueryString != null && lastQueryString.length() > 0) {
-            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.ICE_CREAM_SANDWICH) {
-                MenuItemCompat.expandActionView(searchMenuItem);
-            }
+            searchMenuItem.expandActionView();
```

Comment:
`MenuItemCompat.expandActionView` is deprecated.

Reply:
would be good to test that this doesn't cause any side-effects on Android 4 devices.